db/health/migrate/20230504194752_create_mhx_external_ids.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMhxExternalIds < ActiveRecord::Migration[6.1]
+  def change
+    create_table :mhx_external_ids do |t|
+      t.references :client
+      t.string :identifier
+      t.boolean :valid_id
+      t.timestamps
```

Comment:
```suggestion
      t.references :client, null: false
      t.string :identifier, null: false,
      t.boolean :valid_id, null: false, default: true
      t.timestamps, null: false
```
alternatively, I wonder if `valid_id` should actually be a timestamp, `invalidated_at` and then we can put some logic around how long to wait to check again?

Reply:
Do we want a unique index on `client_id`, or maybe `client_id, identifier`?